waiter/config-shell.edn

```diff
@@ -26,6 +26,12 @@
  ;; Set the port:
  :port #config/env-int "WAITER_PORT"
 
+ ; ---------- Token Storage ----------
+
+ :kv-config {:kind :persistent
+             :persistent {:factory-fn waiter.kv/new-persistent-kv-store
+                          :target-file "kv-store.bin"}}
+
```

Comment:
Does this make the shell scheduler in travis actually run with this? I'm curious if all of the integration tests pass.

Reply:
I will make the composite scheduler run with this, that's the file that is used.